src/OrbitQt/orbitmainwindow.ui

```diff
@@ -522,6 +524,15 @@ QPushButton:disabled {
     <string>Open User Data Directory...</string>
    </property>
   </action>
+  <action name="actionOpenAppDataDirectory">
+   <property name="icon">
+    <iconset resource="../../icons/orbiticons.qrc">
+     <normaloff>:/actions/folder</normaloff>:/actions/folder</iconset>
+   </property>
+   <property name="text">
+    <string>Open App Data Directory...</string>
+   </property>
+  </action>
```

Comment:
What about adding a tooltip to this action telling the user more details like this used to be the old location for captures and presets?

Reply:
I think that they are quite hidden, but they also can't hurt, so I added tooltips for both actions. Let me know what you think of them.

Reply:
Perfect. Thanks a lot!